Style fixes, merge from master

TheGiraffe3 opened the pull request:
Merges master into your branch.
Does style stuff in the files.
Fixes a typo in the ships and adds cloaking functionality to the Vujlet, because the description says it has it. The cloaking ability is not carried over into the Naive version.

tibetiroka committed:
fix(typo): Fix excess empty lines at the end of files (#10294)

bene-dictator committed:
fix(content): Add "Pirate Basics" outfitter to all pirate outfitters, reimplement "Pirate Outfits" outfitter for plugin compatibility (#10288)

TomGoodIdea committed:
feat(enhancement): Custom scan sounds (#10240)

TomGoodIdea committed:
feat(enhancement): `<model>` replacement for model name of the flagship (#10232)

petervdmeer committed:
refactor(conditions): Remove unused Has functionality for derived conditions (#10216)

TheGiraffe3 committed:
refactor(style): Standardize line breaks in the human/variants.txt file (#10291)

2 lines between variants of the same base model, 3 lines between different base models

TheGiraffe3 committed:
fix(content): The Pug planets in Deneb can no longer be bribed (#10210)

TheGiraffe3 committed:
Update successor ships.txt

TheGiraffe3 committed:
Merge branch 'endless-sky:master' into patch-1

TheGiraffe3 committed:
Update successor ships.txt

Typo fix.
Add cloaking.

TheGiraffe3 committed:
Update successor ships.txt

Whoops.

TheGiraffe3 committed:
Fix checks?

TheGiraffe3 committed:
Now?

TheGiraffe3 commented:
Well, the Content Style check is no longer failing, so no checks should have problems once this is merged.
Ready for review, especially for the cloaking values.

TheGiraffe3 committed:
Update successor weapons.txt

TheGiraffe3 committed:
Update successor outfits.txt

TheGiraffe3 committed:
Update successor variants.txt

TheGiraffe3 committed:
Alphabetical order

TheGiraffe3 commented:
Oh, the checks did work.

TheGiraffe3 commented on data/successors/successor variants.txt:
For this file, please look at the way it is in the actual branch, not the diff, which is quite confused.

TheGiraffe3 commented on license.txt:
Ignore all changes below this, they are merging master in.

TheGiraffe3 commented on data/coalition/coalition.txt:
Ignore everything until the successors files, this is for merging master in.

TheGiraffe3 committed:
Update successor events.txt

TheGiraffe3 committed:
Update successor fleets.txt

TheGiraffe3 committed:
Update successor hails.txt

TheGiraffe3 committed:
Update successor jobs.txt

TheGiraffe3 commented:
And yes, it would definitely be wise to squash this, I'm afraid I will have messed history up a lot otherwise.

TheGiraffe3 committed:
Update successor news.txt

TheGiraffe3 committed:
Update successor side missions.txt

TheGiraffe3 committed:
Update successors.txt

TheGiraffe3 committed:
Update successors.txt

Alphabetical I

TheGiraffe3 committed:
Alphabetical II

TheGiraffe3 committed:
Alphabetical III

TheGiraffe3 committed:
Alphabetical IV

TheGiraffe3 commented:
I can't finish the alphabetical stuff right now, so you can either merge it as is, change it yourself, or leave it, your choice.

Daeridanii1 commented on data/successors/successor ships.txt:
```diff
@@ -979,6 +1018,9 @@ ship "Vujlet"
 		"hull repair rate" 0.875
 		"hull energy" 1.27
 		"hull heat" 1.27
+		"cloak" .05
+		"cloaking fuel" .09
+		"cloaking energy" .75
```
```suggestion
```
This makes sense given the description, but my plans for cloaking are that it's something the Successors might rediscover rather than something they start out with. I'll make a note to change the Vujlet's description to make its lack of cloaking ability less ambiguous.

Daeridanii1 commented on data/successors/successor ships.txt:
```diff
@@ -1039,7 +1082,12 @@ ship "Vujlet"
+
 
 ship "Vujlet" "Vujlet (Naive)"
+	add attributes
+		"cloak" -.05
+		"cloaking fuel" -.09
+		"cloaking energy" -.75
```
```suggestion
```

Daeridanii1 reviewed (commented):
Apart from the cloaking, which I've noted, this looks great to me. 
Thank you very much for your PRs and reviews; you've put a ton of work into getting the details right and I very much appreciate it.

Daeridanii1 committed:
Remove cloaking from Vujlet

Apologies for not letting you do this yourself, but I wanted to merge this sooner rather than later to keep the branches from diverging too much.

TheGiraffe3 commented:
> Apologies for not letting you do this yourself, but I wanted to merge this sooner rather than later to keep the branches from diverging too much.

It's fine, thanks.
